app/playbooks/deploy.d/hosted-loadbalancer/hosted-loadbalancer.yml

```diff
@@ -1,7 +1,7 @@
 - name: install self-hosted load balancer
```

Comment:
It would probably be better to remove the hosted-loadbalancer code. It is legacy and probably wouldn't work anymore.

rm -rf app/playbooks/deploy.d/hosted-loadbalancer

Reply:
I can see that there are references for load balancer in the destroy and install scripts - do they reference _that_ load balancer or is that another thing?

Reply:
This is something else. Let's keep it in for now so we can go ahead and merge.